newswires/client/src/ItemData.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearch } from './context/SearchContext.tsx';
+import { Item } from './Item';
+import { pandaFetch } from './panda-session';
+import { type WireData, WireDataSchema } from './sharedTypes';
+import { paramsToQuerystring } from './urlState';
+
+export const ItemData = ({ id }: { id: string }) => {
+	const { handleDeselectItem, config } = useSearch();
+
+	const [itemData, setItemData] = useState<WireData | undefined>(undefined);
+	const [error, setError] = useState<string | undefined>(undefined);
+
+	const maybeSearchParams = useMemo(() => {
```

Comment:
Search params are not needed to retrieve a single story?

Reply:
good point!